bin/clinvar_jsons/extract_like_path_path.py

```diff
@@ -0,0 +1,51 @@
+import argparse
```

Comment:
The name of the script would be more descriptive if renamed to something like `extract_pathogenic_and_likely_pathogenic_variants.py`.